lib/google_drive/config.rb

```diff
@@ -0,0 +1,36 @@
+require 'json'
+
+module GoogleDrive
+  class Config
+    #:nodoc:
```

Comment:
This must be in the end of the previous line, I guess:

    class Config #:nodoc: